feat: Adding `<Logo/>` and `<Tags/>` components with updated colors too

rogeruiz opened the pull request:
- [x] **feat: Refactoring Directory/Logo into component**
- [x] **fix: Updating colors**
- [x] **feat: Adding new Directory/Tags component**

jvmontes reviewed (changes requested):
let's get aligned with the colors, but i love what u did with the Logo and Tags components!

rogeruiz commented on src/app/globals.css:
```diff
@@ -61,6 +66,11 @@
+    --brand-color: #0a131f;
+    --primary-color: #315dc4;
+    --secondary-color: #367dbf;
+    --accent-color: #dba41a;
```
I used a tool called Coolers.co which allows me to shade colors down to get a dark mode equivalent

rogeruiz commented on src/app/globals.css:
```diff
@@ -30,6 +30,11 @@
+  --brand-color: #12263a;
+  --primary-color: #2d56b2;
+  --secondary-color: #8fb8de;
+  --accent-color: #f0d07d;
```
I can change those and the dark colors into HSL later today.

rogeruiz replied:
Also CSS variables can hold arbitrary strings, I'd like for the name of the variable to hold the type of value it is. But we don't need to include the word color in the Tailwind configuration.

rogeruiz commented:
@jvmontes Okay I've updated all the colors to use `hsl()` and moved the call to that function in to the variable declaration rather than where we're referencing the variable. This way we can modify the colors from `hsl()` if we need to. If there's more feedback about colors, I'd feel better to change things in other PRs rather than in this PR. I still need this to be stable before I can get started on the Org Details page again.

jvmontes commented on tailwind.config.ts:
```diff
@@ -40,52 +40,58 @@ const config: Config = {
+        brand: "var(--brand)",
+        accent: "var(--accent)",
+        primary: "var(--primary)",
+        // secondary: "var(--secondary)",
```
nitpick but we could probably uncomment this even if it isn't used.

jvmontes reviewed (approved):
Looks good! Approved 👍🏼
